entrypoint.sh

```diff
@@ -19,6 +18,23 @@ HIDE_SHARES="${HIDE_SHARES:-no}"
 
 # common functions
 set_password() {
+  # check PASSWORD and PASSWORD_FILE are both not set
```

Comment:
I would simplify this and replace `! -z` with `-n`:
```
# check PASSWORD and PASSWORD_FILE are both not set
if [ -n "${PASSWORD}" ] && [ -n "${PASSWORD_FILE}" ]
then
  echo "ERROR: PASSWORD and PASSWORD_FILE can not both be set. Please choose 1"
  exit 1
fi
```

Reply:
Done. Thank you